client/reader/site-stream/featured.jsx

```diff
@@ -23,8 +25,35 @@ export default React.createClass( {
 
 	getStateFromStores( store ) {
 		store = store || this.props.store;
+
+		let postKeys = store.get();
+		let posts = postKeys.map( postKey => {
+			let post = FeedPostStore.get( postKey );
+			if ( ! post || post._state === 'minimal' ) {
+				FeedPostStoreActions.fetchPost( postKey );
+				post._state = 'pending';
```

Comment:
Ah, I see, in the old render method we stash and mutate the post state on a local variable, so we don't change the actual post object. maybe we can add that to the new object that's coming out of the map?